README.rst

```diff
@@ -17,11 +17,11 @@
 
 **Data Version Control** or **DVC** is a command line tool and `VS Code Extension <https://marketplace.visualstudio.com/items?itemName=Iterative.dvc>`_ to help you develop reproducible machine learning projects:
 
-#. **Version** your data and models. Store them in your cloud storage but keep
-   their version info in your Git repo.
+#. **Version** your data and models.
+   Store them in your cloud storage but keep their version info in your Git repo.
```

Comment:
what is the purpose of this change? who many of changes are cosmetic in this PR, what should we actually review? Can we split cosmetic, things related to VS Code and other improvements? If needed do them one by one - there is no rush now.

Reply:
Because of formatting, to review is best to see the the README in this branch as mentioned in the description 🙂

https://github.com/iterative/dvc/tree/readme/how-it-works#readme

There's nothing related to VS Code in this PR, it's about

> - should probably move this (**How it works**) to AFTER the **Quick start** sections
- it's missing info about Experiment Management
- Extra: consistent formatting (1 sentence = 1 line)

I can separate the last point if needed but I'm not sure what formatting we want. Is this one OK @dberenbaum ? 1 sentence = 1 line. Or should it be a fix col width? Currently it's an inconsistent mix of things.

Reply:
> I can separate the last point if needed but I'm not sure what formatting we want. Is this one OK https://github.com/orgs/iterative/teams/dvc ? (1 sentence = 1 line). Or should it be a fix col width? Currently it's an inconsistent mix of things.

yes, let's separate and do it later as a completely separate PR please

Reply:
> * move this (**How it works**) to AFTER the **Quick start** sections
> 
> * it's missing info about Experiment Management

👍 👍 

> * Extra: consistent formatting (1 sentence = 1 line)

Not sure I understand this one. I don't think changes like the one above make any visible difference in the readme (it will still show the whole bullet as one line even with the line break), so what is the point?

Reply:
> I don't think changes like the one above make any visible difference in the readme ... so what is the point?

Just markup code consistency. Currently there's a mix of lines wrapped at different column lengths and some that don't seem to have any limit.